### PR TITLE
Alicia/wcf docker temp

### PR DESCRIPTION
**Pull request by Lxiamail**
To optimize WCF docker support experience in VS, update wcf docker image based on aspnet base image, which will be used by default for IIS hosted scenario (WCF service application project).

### CODE REVIEW

File: 4.6.2/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,8 @@
-FROM microsoft/dotnet-framework:4.6.2
+FROM microsoft/aspnet:4.6.2
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```
Comment (zhenlan) [inline 4.6.2/Dockerfile]: Can you please remind me again why this is necessary?

File: 4.6.2/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,8 @@
...
-Add-WindowsFeature NET-WCF-TCP-Activation45; \
+RUN Add-WindowsFeature NET-WCF-TCP-Activation45; \
 Add-WindowsFeature NET-WCF-HTTP-Activation45; \
-Add-WindowsFeature Web-WebSockets
-
-ADD ServiceMonitor.exe /ServiceMonitor.exe
-
-EXPOSE 80
-
-ENTRYPOINT ["C:\\ServiceMonitor.exe", "w3svc"]
```
Comment (zhenlan) [inline 4.6.2/Dockerfile]: We should remove ServiceMonitor.exe from the repo too.

File: README.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 ## Supported tags and respective `Dockerfile` links
 
-* 4.7-windowsservercore-10.0.14393.1358, latest ([windowsservercore/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.7/Dockerfile))
-* 4.6.2-windowsservercore-10.0.14393.1358 ([windowsservercore/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.6.2/Dockerfile))
+* 4.7-windowsservercore-10.0.14393.1715, latest ([windowsservercore/Dockerfile](https://github.com/Microsoft/wcf-docker/blob/master/4.7/Dockerfile))
```
Comment (zhenlan) [inline README.md]: Is there a way we don't have to update readme.md every time we have a new build? For example, we can add a link to docker hub for people to find the supported tags. Docker hub is the place for the ultimate truth anyway.

File: 4.7/Dockerfile [hunk 4]
```diff
@@ -1,15 +1,8 @@
...
-Add-WindowsFeature NET-WCF-TCP-Activation45; \
+RUN Add-WindowsFeature NET-WCF-TCP-Activation45; \
 Add-WindowsFeature NET-WCF-HTTP-Activation45; \
-Add-WindowsFeature Web-WebSockets
-
-ADD ServiceMonitor.exe /ServiceMonitor.exe
-
-EXPOSE 80
-
-ENTRYPOINT ["C:\\ServiceMonitor.exe", "w3svc"]
```
Comment (zhenlan) [inline 4.7/Dockerfile]: Remove ServiceMonitor.exe from the repo.

File: 4.6.2/Dockerfile (hunk 1 above)
Comment (Lxiamail) [reply 4.6.2/Dockerfile]: This is for installing WAS WCF components on top of aspnet configuration.

File: README.md (hunk 3 above)
Comment (Lxiamail) [reply README.md]: Good catch.

File: 4.6.2/Dockerfile (hunk 2 above)
Comment (Lxiamail) [reply 4.6.2/Dockerfile]: Good catch

File: 4.6.2/Dockerfile.old [hunk 5]
```diff
@@ -0,0 +1,15 @@
+FROM microsoft/dotnet-framework:4.6.2
```
Comment (zhenlan) [inline 4.6.2/Dockerfile.old]: Why do we add this Dockerfile.old file? Forgot to exclude from the committing? :)

File: 4.6.2/Dockerfile [hunk 6]
```diff
@@ -1,15 +1,8 @@
...
+FROM microsoft/aspnet:4.6.2
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 # install WCF services required components.
-RUN Add-WindowsFeature Web-Server; \
-Add-WindowsFeature NET-WCF-TCP-Activation45; \
+RUN Add-WindowsFeature NET-WCF-TCP-Activation45; \
 Add-WindowsFeature NET-WCF-HTTP-Activation45; \
-Add-WindowsFeature Web-WebSockets
```
Comment (zhenlan) [inline 4.6.2/Dockerfile]: Why remove websockets?

Comment (zhenlan) [reply 4.6.2/Dockerfile]: Ah, forget about it. It's how GitHub shows the diff.

File: 4.6.2/Dockerfile.old (hunk 5 above)
Comment (Lxiamail) [reply 4.6.2/Dockerfile.old]: :) removed

**Pull request merged**